### DISCUSSION TITLE
v2.13.0

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#91] [Announcements]
Scythe Anticheat is now at version v2.13.0!

Supported Minecraft Versions: 1.19.80, 1.19.81

# Changelog
- Add support for 1.19.80
- Added new config option to send an invalid command message if an invalid scythe-command is sent
- Fixed BadEnchant checks not working
- Lots of improvements to chat rank
  - Removed <> from player name
  - Added config setting to change tag bracket color
  - Added config setting to change the tag color
  - Added config setting to change player name color
- Bug fixes
- Performance optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/MrDiamond64/Scythe-AntiCheat/releases/tag/'>v2.13.0</a>.</em>

**Comment by DaddyGhosti** [comment]
heyy i cant download the A.C i cant find the .mcpack for my advice
